src/Workleap.Extensions.OpenAPI/RequiredType/SetPropertyAsRequiredTypeFilter.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Workleap.Extensions.OpenAPI.RequiredType;
+
+internal sealed class SetPropertyAsRequiredTypeFilter : ISchemaFilter
```

Comment:
We could add a comment that explain it is to be used with the SupportNonNullableReferenceTypes that will use the nullable C# feature to set nullable.